### ISSUE TITLE
streamdeck-ui, seems stale, lets decide an initial course of action to keep the project alive.

### ISSUE THREAD

**Issue by 4s3ti** [timothycrosley/streamdeck-ui#357]
As per discussion at: https://github.com/timothycrosley/streamdeck-ui/pull/354  lets start to take some initial actions. 

@dodgyrabbit  you seem to be one of the biggest contributors, do you have any suggestions?

Let's wait a couple of days (I'd say until Tuesday or Wednesday) and see if @dodgyrabbit  shows up as well and shares his view. 

Please react here with where you would prefer to have some sort of chat where we can have a more active talk, and find some maintainers for this.

React here with: 
👀  : Discord
🚀 : Slack
🎉 : Matrix
❤️ : Other, Comment which. 

Also, react with 👍  If you are whiling to assist in keeping this project alive, and add a comment with what role you would like to assume.

Feel free to throw away any more suggestions if you feel that I am neglecting something.

**Comment by Data-007** [comment]
Hi
Some sort of **Discord** option I would favor myself.
I would also like to add #208 to your list above. It is most comprehensive suggestion, by @dodgyrabbit 
I don't know  what exactly is proposed with #244 I will have to investigate. 

Im first and foremost a Alpha/Beta tester  I can read Python and participate that way.
I test work on an Arch based system.
 
 Im here to try and breath live into the project where ever it goes.
 --
DaTA-007

**Comment by StormyIceLeopard** [comment]
I would like to see development of this program to continue.

**Comment by Narcodepsy** [comment]
Discord works for comms for me. (can use others just not as familar with them)

Happy to help test (debian) user mainly, but have most distros snapshots ready in VMs.
(streamdeck mk2)

Like this project, compared to the other i have seen its already in the best condition so keeping it moving forward makes the most sense.

**Comment by StormyIceLeopard** [comment]
I can help test too
I use Manjaro

**Comment by Data-007** [comment]
https://discord.gg/ZCZesnEj4

**Comment by sftim** [comment]
I'm willing to help with docs, a bit. I hope we can help people contribute even if they rarely use Discord.

**Comment by 4s3ti** [comment]
> I'm willing to help with docs, a bit. I hope we can help people contribute even if they rarely use Discord.

no need for discord, just a place where can exchange info a bit more quickly. 
anyhow, if by Wednesday there's no info from anyone originally involved in the project ill have a place to move on going forward. till then, let's wait and hold on.

**Comment by morrolinux** [comment]
I use this program every day and I even forked it to do some dirty but useful customization which could also potentially be useful for someone else. I can promise you nothing but I'm certainly willing to share my thoughts and better integrate code changes if needed. PS: Discord is fine for me.

**Comment by Data-007** [comment]
Discord server is up just use the link in the post's above.
We would love to have you, in fact here is the link https://discord.gg/ZCZesnEj4

**Comment by Data-007** [comment]
> I like Discord I can help test too I use Manjaro

@StormyIceLeopard  come Join us on Discord https://discord.gg/ZCZesnEj4

**Comment by 4s3ti** [comment]
@irthomasthomas GH will be the de facto place, but for now till we have a home is a good place to get some stuff going. 

I am currently preparing a home for the fork

**Comment by Data-007** [comment]
> Isn't github the better place for this discussion? Maybe I'm just thick but I have a hard time finding things and following threads in discord. Seems more suited for organizing ephemeral events, more than recording peoples thoughts on the future of a github project?

You make some excellent points, Can you tell us here in this thread then your view on the matters at hand?
We are all discussing things on discord and your right for some it maybe hard to follow, but all view's matter!
All I can say is that things have been discussed at grate length on discord to much to copy and paste here.

Let me explain that when you follow the link for discord you land in the lobby and are given a role automatically, you can
then just join that one text room and the lobby and join the discussion. 
I hope that helps you if you want to talk on GitHub about your views fine we/I am happy to talk.

We do not want to exclude anyone.

--

Data-007

**Comment by dhtseany** [comment]
I'm just getting up to speed with this thread however if this really is the path forward for the project then yes, I'll shift the source for the Arch package I maintain to the new repo in the coming weeks, for now I'll simply patch the package to work with python-pillow 10.

In regards to communication I too am against chat systems for publishing important information like commits or known issues or for users seeking support. Of course any dev team needs private communication channels to work on stuff together but when it comes to discussing problems with the app that impacts everyone sticking with the Github Issues tabs (_or similar feature on similar services_) is how I think you should keep things going moving forward because it's easy enough for search engines to scrape and for non-chat-system-users like myself to find when they're having problems. Plus I already have 5 chat apps on my phone, none of which are Discord, and I don't have time to watch a 6th.

I've built a side-gig live streaming local governments and the Streamdeck is central to my setup, and streamdeck-ui lets me have my system linux-based, so I have my own interests in keeping this app going. My system can be available as needed for testing as I'm a user that's against using pip for app installation so unlike a lot of other users I'll be packaging it natively, which in many case shows problems differently than how pip users see them.

**Comment by Data-007** [comment]
> I'm just getting up to speed with this thread however if this really is the path forward for the project then yes, I'll shift the ...

Its early days wink wink but we sort off have a new home already, and I am sure it WILL have issues and pull requests
Just like here. I would not panic :P 

It maybe only a very short while away.

--
Data-007

**Comment by 4s3ti** [comment]
Hi all, 

New repository is ready for usage, and you are all welcome to contribute with whatever you can. 

here is it: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui

Discord is no longer the primary point of contact, all topis should be discussed either in [issues section](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/issues) or in [discussions](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions)

Please be sure to read the https://github.com/streamdeck-linux-gui/streamdeck-linux-gui#general-guidelines as some stuff as changed in the way the repository will function, much more relies on automation rather than manual steps taken by the maintainers.

Please also be sure to read the Documentation: https://streamdeck-linux-gui.github.io/streamdeck-linux-gui/ which has suffered some changes as well. 

Version 3.0.0 has been released, There are no breaking changes however I felt that it would be good to bump the major version to signal the change. 

[here](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/2) you can find the announcement, and future announcements shall be posted there. 

@dhtseany hope this answers most of your concerns. I'd be glad to assist you with whatever you need to keep the AUR package healthy and add you as maintainer if you wish. Also, feel free to reach me out in any of your chat applications, I am sure you can find me in one of them or worst case scenario via e-mail :)

If the original core devs return, they will have in their mailbox an invite to the organization with full ownership over the project!



Hope this answers all of your questions, and the new home is structured and functional enough to keep going forward!

**Comment by RemcoSchrijver** [comment]
I wrote up a small migration guide [here](https://writerit.nl/productivity/streamdeck/migrating-from-streamdeck_ui-to-streamdeck-linux-gui/) for those that want to move but didn't do it yet.
